src/mtcp/mtcp_restart.c

```diff
@@ -793,18 +794,31 @@ unmap_memory_areas_and_restore_vdso(RestoreInfo *rinfo)
   }
 
   // Check for overlap between newer and older vDSO/vvar sections.
-  if (doAreasOverlap(vdsoStart, vdsoEnd - vdsoStart,
+  if (
+#if 0
```

Comment:
@gc00 Do we need this code (\#if 0 ... \#endif) now that we have a fix for overlapping regions?